javalib/src/main/scala/java/time/temporal/ChronoField.scala

```diff
@@ -1,9 +1,9 @@
 package java.time.temporal
 
 final class ChronoField private (name: String, ordinal: Int,
-    _range: ValueRange, dateBased: Boolean, baseUnit: ChronoUnit,
-    rangeUnit: ChronoUnit) extends Enum[ChronoField](name, ordinal)
-    with TemporalField {
+    _range: ValueRange, dateBased: Boolean, timeBased: Boolean,
```

Comment:
Rater than adding a new field you should repurpose the `dateBased` one as a flag container. You could have `basedOnFlags: Int` instead and then use them as:
```scala
private final val isTimeBasedFlag = 1
private final val isDateBasedFlag = 2

def isTimeBased(): Boolean = (basedOnFlags & isTimeBasedFlag) != 0

... ValueRange.of(0, 86399999), isTimeBasedFlag, MILLIS, DAYS)
```